### ISSUE TITLE
Path is missing from JSESSIONID cookie in 12 beta 0

### ISSUE THREAD

**Issue by Artur-** [eclipse/jetty.project#9459]
**Jetty version(s)**
12.0.0.beta0 / ee10

**Java version/vendor**
openjdk version "19.0.1" 2022-10-18

**OS type/version**
mac os

**Description**
One of the tests in Vaadin first requests

http://localhost:8888/web-component/themed-component.js

and then later on

http://localhost:8888/

Both of the requests use the session.

Now with Jetty 12 the response to the first request contains
```
Set-Cookie: JSESSIONID=node01esb8me0d0pxv1ecc7ivw1d4uh1.node0
```
causing Chrome to store the cookie with the path `/web-component`

It is thus not passed with the second request to http://localhost:8888/.

Now one of two things can happen, depending on some external circumstances like what is cached and network loading order
1. The request to / has no cookie, Vaadin expects it to have and an error is printed
2. The request to / creates a new session and sets a new session id cookie

If a new session cookie is set, then the following request to / includes two session cookies and Jetty directly returns "bad request"

With Jetty 11 and earlier the first request response has
```
Set-Cookie: JSESSIONID=node0j20fd8h8crx91052t9avzbax11.node0; Path=/
```
and thus no further session id cookies are set and no more sessions created

**Comment by janbartel** [comment]
@Artur- when you ran in jetty-11, did you set the session cookie config at all? Either via context init params or a `cookie-config` in `web.xml`?

**Comment by Artur-** [comment]
I don't think so

**Comment by janbartel** [comment]
Closed via PR #9477
